setup.py

```diff
@@ -116,13 +117,59 @@ def get_setup_kwargs(raw=False):
     return kwargs
 
 
+def avoid_common_setup_errors(raw=False):
+    # This runs before the setup, trying to catch common setup errors
+    try:
+        get_distribution("telegram")
```

Comment:
why can't I find any docs on what this does though

Reply:
https://setuptools.pypa.io/en/latest/pkg_resources.html#getting-or-creating-distributions

Basically checks for anything that was installed via `pip install telegram`